src/components/ShortcutTooltip/ShortcutTooltip.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import { Popup } from 'decentraland-ui'
+import { ShortcutCombination, SimpleShortcut, ShortcutAlternative, ShortcutDefinition } from 'modules/keyboard/types'
+import { mapLabel } from 'modules/keyboard/utils'
+import Chip from 'components/Chip'
+import { Props } from './ShortcutTooltip.types'
+import './ShortcutTooltip.css'
+
+export default class ShortcutTooltip extends React.PureComponent<Props> {
+  static defaultProps = {
+    className: ''
+  }
+
+  renderCombination = (shortcut: ShortcutCombination) => {
+    let out: JSX.Element[] = []
+
+    for (let i = 0; i < shortcut.value.length; i++) {
+      out.push(<Chip text={mapLabel(shortcut.value[i])} />)
+
+      if (i !== shortcut.value.length - 1) {
+        out.push(<span className="plus">+</span>)
+      }
+    }
+
+    return out
+  }
+
+  renderSimple = (shortcut: SimpleShortcut) => {
+    return <Chip text={mapLabel(shortcut.value)} />
+  }
+
+  renderAlternative = (shortcut: ShortcutAlternative) => {
```

Comment:
`renderCombination`, `renderSimple` and `renderAlternative` could be moved to a util/helper and be reused in `ShortcutModal` and `ShortcutTooltip` since they don't use `this`